src/Tickets/Seating/Frontend.php

```diff
@@ -64,6 +68,30 @@ protected function do_register(): void {
 		add_filter( 'tribe_template_pre_html:tickets/v2/tickets', [ $this, 'print_tickets_block' ], 10, 5 );
 
 		// @todo regsiter front-end Assets here.
+		
+		// Register the front-end CSS.
+		Asset::add(
+			'tec-tickets-seating-frontend-css',
```

Comment:
Register the CSS after the JS.
The convention is:
* no postfix (e.g. no `-js`) for the JS file.
* `-style` postfix for the CSS file.